lib/lotus/entity.rb

```diff
@@ -138,6 +138,17 @@ module ClassMethods
       #
       #   User.attributes => #<Set: {:id, :name}>
       #   DeletedUser.attributes => #<Set: {:id, :name, :deleted_at}>
+      #
+      # @example Add timestamps
+      #   require 'lotus/model'
+      #
+      #   class User
+      #     include Lotus::Entity
+      #     attributes :timestamps
```

Comment:
What do you think about having a separate method, for example `add_timestamps` instead? IMHO I think we should allow users to have attribute with naming 'timestamps'.